src/trip.c

```diff
@@ -284,3 +285,59 @@ Trips* GetTrips(CURL* curl, const char* origin, const char* destination) {
 
   return trips;
 }
+
+void CoordinatesForStations(CURL* const curl) {
+  // This is a reference to a journey detail. More logic need to be implemented
+  const char* k_ref =
+      "http://webapp.rejseplanen.dk/bin//rest.exe/"
+      "journeyDetail?ref=171774%2F109037%2F608648%2F247066%2F86%3Fdate%3D15.12."
+      "23%26station_evaId%3D8600798%26format%3Djson";
+
+  Response response = {NULL, 0};
+  DoRequest(curl, k_ref, &response);
```

Comment:
@TexWO This won't work, since `DoRequest` prepends the API base URL to any request. So the second parameter should start with `journeyDetail...`.